ci/deploy/gather-indev/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "gather-indev",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {},
+  "dependencies": {
+    "@octokit/rest": "^18.12.0",
+    "@types/node": "^16.11.11",
+    "ts-node": "^10.4.0",
+    "typescript": "^4.5.2"
```

Comment:
@types/node and typescript probably should be in `devDependencies`

Reply:
Hmm, probably `ts-node` too then.

But, strangely, when I run `yarn install --prod` without `ts-node` moved to `devDependencies`, I get `typescript` and `@types/node` installed anyway.